### ISSUE TITLE
Add note about DOM when running tests in Node

### ISSUE THREAD

**Issue by jmeas** [babel/generator-babel-boilerplate#234]
The Parsley team [ran into an issue](https://github.com/guillaumepotier/Parsley.js/pull/968#issuecomment-143246388) where they were running tests in Node, but didn't load up a server-side implementation of the DOM, like JSDom. So jQuery blew up.

I should add a note in the README about making sure to load the tools you need for the tests to work on the server! This jQuery use case is prob. a big one.

**Comment by marcandre** [comment]
Maybe commented code in `setup/node.js` would be a good start:
```
// Uncomment the following if your library uses features of the DOM (e.g. a jquery extension),
// and add 'simple-jsdom' to the `devDependencies` of your package.json.
// import simpleJSDom from 'simple-jsdom';
// simpleJSDom.install();
```

**Comment by jmeas** [comment]
Good idea. Care to make a PR?

**Comment by marcandre** [comment]
I indeed should have created a PR, sorry about that. I guess I should also stop complaining about slackers :smirk:

**Comment by jmeas** [comment]
Resolved in #235

**Comment by jmeas** [comment]
Thanks again @marcandre !
